source/_integrations/zwave_js.markdown

```diff
@@ -820,7 +821,7 @@ In addition to the [standard automation trigger data](/docs/automation/templatin
 
 ### `zwave_js.event`
 
-This trigger platform can be used to trigger automations on any Z-Wave JS controller, driver, or node event, including events that may not be handled by Home Assistant automatically. Refer to the linked [Z-Wave JS documentation](https://zwave-js.github.io/node-zwave-js/#/) to learn more about the available events and the data that is sent along with it.
+This trigger platform can be used to trigger automations on any Z-Wave JS adapter, driver, or node event, including events that may not be handled by Home Assistant automatically. Refer to the linked [Z-Wave JS documentation](https://zwave-js.github.io/node-zwave-js/#/) to learn more about the available events and the data that is sent along with it.
```

Comment:
This is actually referring to the [controller events](https://zwave-js.github.io/zwave-js/#/api/controller?id=controller-events), which are also named like that in code. I don't think we're going to rename this.